src/hooks/useMousetrap.ts

```diff
@@ -54,10 +54,12 @@ export const useMousetrap: ReactHook<UseMousetrapProps, void> = ({
   callback,
   action,
   withGlobalBind = false,
-
-  // @ts-ignore
-  bindRef,
+  ...rest
 }) => {
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  const bindRef = rest?.bindRef;
+
```

Comment:
What's the reason for this change?

Reply:
Forgot about it. Reverted back in https://github.com/sparkletown/sparkle/pull/1977/commits/fdf67dbe5bdeeaf6d08d0c682dace7fdb4e3a761